### PR TITLE
Added site logo to the footer of the site 

### LINKED ISSUE #302: Add website logo in the footer
Add the logo inside the box

Screenshot (7)

Use this logo -> Voters-Registration-Portal/Images/Logo/logo1.jpg

https://github.com/QAZIMAAZARSHAD/Voters-Registration-Portal/blob/master/Images/Logo/logo1.jpg

### PR DESCRIPTION
**Pull request by Croooook**
#### Describe the changes you've made
Added the website logo in the footer region of the website under footer div class 4 in HTML code for feedbackform.html.

#### Describe if there is any unusual behavior of your code(Write `NA` if there isn't)
NA

#### Test plan (OPTIONAL)
A good test plan should give instructions that someone else can easily follow.
How someone can test your code?

#### Checklist

- [x] I have mentioned related issue number in PR description.
- [x] I have added screenshots and related demo links of my changes..
Screenshot (248)

- [ ] The title of my pull request is a short description of the requested changes.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] Open Source Program names DevIncept

### CODE REVIEW

**Comment by nlok5923** [comment]
@Croooook  the logo is currently shifted right ward w.r.t the footer links section.
image
try to make it align with the link section of footer to bring uniformity.

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (nlok5923) [inline package-lock.json:1]: delete this file. it is not necessary

**Review by nlok5923 — CHANGES_REQUESTED** [review]
Thanks PTAL !!

**Comment by nlok5923** [comment]
@Croooook also please correctly link issue with this pull request.

File: package-lock.json (hunk 1 above)
Comment (Croooook) [reply package-lock.json]: Yeah, I deleted the extra files in the new commit .. and also changed the alignment of the image with respect to the link section of footer.. pls can you check it out again?

Comment (nlok5923) [reply package-lock.json]: please provide a deployment link to test deployment also change the inline css to classbased css.

Comment (Croooook) [reply package-lock.json]: deployment link i tried but .. netlify was not providing link for the branch files ... instead it was creating link for the master branch for my repo of the clone project..

**Comment by QAZIMAAZARSHAD** [comment]
@Croooook use GitHub pages.

File: package-lock.json (hunk 1 above)
Comment (Croooook) [reply package-lock.json]: let me try again.. though i did this yesterday but still no change .. it was not taking my branches into the site link ..

**Comment by Croooook** [comment]
> @Croooook use GitHub pages.

for creating link?

**Comment by QAZIMAAZARSHAD** [comment]
> > @Croooook use GitHub pages.
> 
> for creating link?

Yes

**Comment by QAZIMAAZARSHAD** [comment]
Resolve conflicts also

**Comment by Croooook** [comment]
@nlok5923  can  pls help me solving the conflicts ...
"<<<<<<< branch1
=======
>>>>>>> master "
 bcz of this change in the source code there are conflicts coming .. and when i am copy pasting the same in my local code and then commit ..push.. the same issue is added somewhere else in to code .. hence becoming am infinite loop for me.. can u pls help me solve this?

**Comment by nlok5923** [comment]
@Croooook  for resolving conflicts you just need to keep the incoming changes along with the changes done in this pr that's it you can click on resolve conflicts button and then keep the necessary incoming changes and the changes done by you once done mark it as resolved.

**Comment by nlok5923** [comment]
@Croooook  i had reviewed the code and the conflicts are no code conflicts so i resolved it on your side. Now do one thing just pull the commit made by resolving conflict and then deploy your code and provide us deployment link for testing purpose.

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,3 +1,16 @@
 {
-    "liveServer.settings.port": 5501
+  "liveServer.settings.port": 5501,
+  "C_Cpp_Runner.warnings": [
+    "-Wall",
+    "-Wextra",
+    "-Wpedantic"
+  ],
+  "C_Cpp_Runner.compilerArgs": [],
+  "C_Cpp_Runner.includePaths": [],
```
Comment (nlok5923) [inline .vscode/settings.json:9]: unnecessary changes 🤕

**Review by nlok5923 — CHANGES_REQUESTED** [review]
Thanks PTAL !!

File: .vscode/settings.json (hunk 2 above)
Comment (Croooook) [reply .vscode/settings.json:9]: yaah amended it!

**Comment by Croooook** [comment]
https://croooook.github.io/Voters-Registration-Portal/ 

deloyed the link to test the changes

**Comment by QAZIMAAZARSHAD** [comment]
@Croooook lower the logo downward 1cm more.

File: .vscode/settings.json (hunk 2 above)
Comment (nlok5923) [reply .vscode/settings.json:9]: you didn't removed it why ?

**Review by nlok5923 — CHANGES_REQUESTED** [review]
PTAL

**Comment by QAZIMAAZARSHAD** [comment]
Also, curve the corners a bit like in the header.

**Comment by nlok5923** [comment]
> @Croooook lower the logo downward 1cm more.

@QAZIMAAZARSHAD do we need the hover effect as the logo in header has ?

**Comment by Croooook** [comment]
> > @Croooook lower the logo downward 1cm more.
> 
> @QAZIMAAZARSHAD do we need the hover effect as the logo in header has ?

I also thought about it. But hovering the logo in the footer section was not looking good.

**Comment by QAZIMAAZARSHAD** [comment]
> > @Croooook lower the logo downward 1cm more.
> 
> @QAZIMAAZARSHAD do we need the hover effect as the logo in header has ?

Yes, @Croooook this will be great.

But the flip logo feature in the header is wrong.
The logo should flip in its place.

**Comment by Croooook** [comment]
> > > @Croooook lower the logo downward 1cm more.
> > 
> > 
> > @QAZIMAAZARSHAD do we need the hover effect as the logo in header has ?
> 
> Yes, @Croooook this will be great.
> 
> But the flip logo feature in the header is wrong.
> The logo should flip in its place.

Ok I will try , but I am not sure I will be able to do it because I am not that skilled in this.

File: .vscode/settings.json (hunk 2 above)
Comment (Croooook) [reply .vscode/settings.json:9]: I think I took it wrong then.. do I have to completely delete this file? or just delete the code which you are specifying above?

Comment (Croooook) [reply .vscode/settings.json:9]: ok I did something .. can you pls check this time I removed it properly ?

Comment (nlok5923) [reply .vscode/settings.json:9]: No it's not correct only this line should be there 
```
{
 "liveServer.settings.port": 5501
}
```

rest lines aren't related to this project

**Review by nlok5923 — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 2 above)
Comment (Croooook) [reply .vscode/settings.json:9]: yaah i did it only I guess..
Screenshot (252)

Comment (nlok5923) [reply .vscode/settings.json]: ya it's done now.

**Comment by nlok5923** [comment]
@QAZIMAAZARSHAD i think this pr is ready to merge now.

**Comment by Croooook** [comment]
> @QAZIMAAZARSHAD i think this pr is ready to merge now.

Can you pls do it .. I need a PR merged today only to show somewhere else for the prrof

**Comment by QAZIMAAZARSHAD** [comment]
@Croooook you have added too much extra space in the footer.

**Comment by Croooook** [comment]
> @Croooook you have added too much extra space in the footer.
ok.. give me 2 min

**Comment by QAZIMAAZARSHAD** [comment]
Screenshot (58)


Don't disturb any other content.

Also, the dark/light toggle button is missing in your website, looks like your repo is not updated.

**Comment by QAZIMAAZARSHAD** [comment]
No, that toggle is fine, I just noticed that on the login page it is not in the footer.

Can you shift it in the footer also?

**Comment by Croooook** [comment]
> Screenshot (58)
> 
> Don't disturb any other content.
> 
> Also, the dark/light toggle button is missing in your website, looks like your repo is not updated.

Screenshot (255)

but it is saying it updated

**Comment by QAZIMAAZARSHAD** [comment]
> No, that toggle is fine, I just noticed that on the login page it is not in the footer.
> 
> Can you shift it in the footer also?

Yaa it is updated.
It was my mistake.


Can you shift the dark/light toggle of the login page in the footer?

**Review by nlok5923 — APPROVED** [review]
Approving this pr as code changes are fine.

**Comment by QAZIMAAZARSHAD** [comment]
@Croooook you have not filled the google form.

Fill ASAP if you want to score points.
